models/book.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const mongoose = require('mongoose');
+const {Schema} = mongoose;
+
+
+const bookSchema = new Schema ({
+  name: { type: String },
+  description: { type: String},
+  status: { type: String},
+  email: {type: String}
+});
```

Comment:
Good job on the first lab!! 👍 

- Double check your schema. The usual format is as follows:

```
  title: String,
  description: String,
  status: String,
  email: String
```